root/usr/lib64/ruby/vendor_ruby/discovery/screen/status.rb

```diff
@@ -68,8 +68,12 @@ def screen_status status = generate_info, active_button = 0
   elsif answer == b_ssh
     :screen_ssh
   elsif answer == b_resend
-    command("rm -f /tmp/discovery-http*")
-    command("systemctl reload discovery-register")
+    if cmdline('BOOTIF')
+      command("rm -f /tmp/discovery-http*")
+      command("systemctl reload discovery-register")
+    else
+      Newt::Screen.win_message("Not supported", "OK", "Resending not possible in PXE-less, reboot and start over.")
+    end
```

Comment:
For technical reasons, we just error out in case user presses this button, I do not want to re-implement whole workflow to be able to resend the facts from GUI now (I don't have parameters which are being passed among "screen_" methods).